modules/administration-guide/partials/proc_using-chectl-to-install-che-on-codeready-containers.adoc

```diff
@@ -15,18 +15,18 @@ This section describes how to install {prod-short} on CodeReady Containers (CRC)
 
 .Procedure
 
-. Initiate the CRC platform:
-
-.. Configure your host machine for CodeReady Containers:
+.. Remove any previous cluster:
 +
 ----
-$ crc setup
+$ crc delete
 ----
 
-.. Remove any previous cluster:
+. Initiate the CRC platform:
+
+.. Configure your host machine for CodeReady Containers:
 +
 ----
-$ crc delete
+$ crc setup
```

Comment:
```suggestion
$ crc setup
$ crc delete
```

Reply:
I would reverse these directions, personally. We want to delete an existing instance if it already exists.

Note that the `crc cleanup` command also exists to ensure that changes made during `crc setup` are, more or less, undone. If you want to **completely** remove a previous instance (or installation in general), the procedure looks like this:

```
$ crc delete
$ crc cleanup
$ crc setup
```

Reply:
fully agree :+1: